scripts/commerce.js

```diff
@@ -293,3 +293,27 @@ export function setJsonLd(data, name) {
   script.dataset.name = name;
   document.head.appendChild(script);
 }
+
+export async function loadErrorPage(code = 404) {
+  const htmlText = await fetch(`/${code}.html`).then((response) => {
+    if (response.ok) {
+      return response.text();
+    }
+    throw new Error(`Error getting ${code} page`);
+  });
+  const parser = new DOMParser();
+  const doc = parser.parseFromString(htmlText, 'text/html');
+  document.body.innerHTML = doc.body.innerHTML;
+  document.head.innerHTML = doc.head.innerHTML;
+  Array.from(doc.head.querySelectorAll('script'))
```

Comment:
Since you already set the head's inner html, I think every script ends up in the head twice that way?

![Screenshot 2024-07-02 at 15 31 24](https://github.com/hlxsites/aem-boilerplate-commerce/assets/24207564/dc6718e5-a967-4ddd-864a-69035f6906ae)

Reply:
That's correct. The ones set by `.innerHTML` are not executed though, only the manually created ones. Wdyt about removing the others before adding the copies?

Reply:
> The ones set by .innerHTML are not executed though

TIL. It would probably make it easier to understand if the scripts are only there once - or if the code had a comment to explain why the scripts are being copied. Either works IMO.

Reply:
I removed the duplicate scripts and additionally added a comment to explain why the `script` tags have to be re-created.